app/src/main/res/drawable/default_avatar.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="#FFFFFF"/>
+
+    <size android:height="35dp"
```

Comment:
I saw that we use 35dp for gravators in `setUpGravator`
```
 // From http://stackoverflow.com/questions/4605527/
            Resources resources = context.getResources();
            float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP,
                    35, resources.getDisplayMetrics());
String url = message.getSender().getAvatarURL() + "&s=" + px;
            Picasso.with(context)
                    .load(url)
                    .placeholder(android.R.drawable.stat_notify_error)
                    .error(android.R.drawable.presence_online)
                    .into(messageHolder.gravatar);
```
Though, I agree, they do look pretty big, I'll make the amends and edit the pr.